blocks/cards/cards.css

```diff
@@ -72,10 +84,84 @@
     align-self: center;
   }
 
+  .block.cards.image-cards {
+    flex-wrap: nowrap;
+  }
+
   .block.cards .card > div:has(.title) {
     display: flex;
     align-self: stretch;
     align-items: center;
     gap: 10px;
   }
 }
+
+.block.cards.image-cards .card {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+}
+
+.block.cards.image-cards .card-img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  overflow: hidden;
+  border-radius: 16px;
+}
+
+.block.cards.image-cards .card-img img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  opacity: 0.9;
+  z-index: -1;
+}
+
+.block.cards.image-cards .card:hover .card-img img {
+  width: 100%;
+  height: 100%;
+  transform: scale(1.2);
```

Comment:
the `zoom-in ``zoom-out` effect needs to be smooth. maybe an `ease-in-out` will help
please check on the original site once.

Reply:
It's just a linear transition, I added it now.